### ISSUE TITLE
No discernible text on the search button

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#205]
I am also getting the same error on the search button as with the social media icons that there is "o discernible text" The suggested fix is:

To solve this violation, you need to:
Fix at least one (1) of these issues:
Element has a value attribute and the value attribute is empty
Element has no value attribute or the value attribute is empty
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"
And
Fix the following:
Element is in tab order and does not have accessible text

**Comment by joelworsham** [comment]
Same as the social media icons (#204). I had screen reader text, but I just added `aria-label` to be more thorough.
